Helpers/CommandConversion.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BrackeysBot.Helpers
+{
+    static class CommandConversion
```

Comment:
Should document what this class does (and its members). And should just generally comment more code that was added.